roboteam_ai/src/utilities/Coach.cpp

```diff
@@ -253,6 +265,55 @@ Vector2 Coach::getFormationPosition(int robotId) {
     return {targetLocationY, 0};
 }
 
+int Coach::getRobotClosestToGoal(bool ourRobot, bool ourGoal) {
+    roboteam_msgs::World_<std::allocator<void>>::_them_type robots = ourRobot ? World::get_world().us : World::get_world().them;
+    Vector2 target = ourGoal ? Field::get_our_goal_center() : Field::get_their_goal_center();
```

Comment:
there is already a function for this in Field

Reply:
moved